### PR TITLE
Added note about exclude includes bin and obj folders

### LINKED ISSUE #10249: Exclude glob is inaccurate
In [Default compilation includes in .NET Core projects](https://docs.microsoft.com/zh-cn/dotnet/core/tools/csproj#default-compilation-includes-in-net-core-projects) section, it says the exclude globs are `**/*.user; **/*.*proj; **/*.sln; **/*.vssscc`.

Actually it also excludes `$(BaseOutputPath)/**` (*bin* **by default**) and `$(BaseIntermediateOutputPath)/**` (*obj* **by default**). src is [here](https://github.com/dotnet/sdk/blob/master/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.DefaultItems.targets#L25).

I also recommend adding a hint that all of them can be reffered by `$(DefaultItemExcludes)` so that customize excluding would be easier.

And I don't know whether the hyphen in *Remove glob* of *None* is by design.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: cb4428fc-215c-78e1-5ff0-fe73f7a08260
* Version Independent ID: 6741b1de-38a4-fbb1-d79d-acc47d549a2d
* Content: [Additions to the csproj format for .NET Core - .NET Core CLI](https://docs.microsoft.com/en-us/dotnet/core/tools/csproj)
* Content Source: [docs/core/tools/csproj.md](https://github.com/dotnet/docs/blob/master/docs/core/tools/csproj.md)
* Product: **dotnet-core**
* GitHub Login: @blackdwarf
* Microsoft Alias: **mairaw**

### PR DESCRIPTION
**Pull request by Thraka**
Fixes #10249

### CODE REVIEW

File: docs/core/tools/csproj.md [hunk 1]
```diff
@@ -39,11 +39,14 @@ The main reason for doing this is to reduce the clutter in your project file. Th
...
+| Element          	| Include glob                           	  | Exclude glob                                     	            | Remove glob             	 |
 |-------------------|-------------------------------------------|---------------------------------------------------------------|----------------------------|
 | Compile          	| \*\*/\*.cs (or other language extensions) | \*\*/\*.user;  \*\*/\*.\*proj;  \*\*/\*.sln;  \*\*/\*.vssscc 	| N/A                     	 |
 | EmbeddedResource 	| \*\*/\*.resx                             	| \*\*/\*.user; \*\*/\*.\*proj; \*\*/\*.sln; \*\*/\*.vssscc     | N/A                     	 |
-| None             	| \*\*/\*                                  	| \*\*/\*.user; \*\*/\*.\*proj; \*\*/\*.sln; \*\*/\*.vssscc     | - \*\*/\*.cs; \*\*/\*.resx |
+| None             	| \*\*/\*                                  	| \*\*/\*.user; \*\*/\*.\*proj; \*\*/\*.sln; \*\*/\*.vssscc     | \*\*/\*.cs; \*\*/\*.resx   |
+
+> [!NOTE]
+> **Exclude glob** always excludes the `./bin` (which is `$(BaseOutputPath)`) and `./obj` (which is `$(BaseIntermediateOutputPath)`) folders. As a whole, all excludes are represented by `$(DefaultItemExcludes)`.
```
Comment (mairaw) [inline docs/core/tools/csproj.md]:
```suggestion
> **Exclude glob** always excludes the `./bin` and `./obj` folders, which are represented by the `$(BaseOutputPath)` and `$(BaseIntermediateOutputPath)` MSBuild properties, respectively. As a whole, all excludes are represented by `$(DefaultItemExcludes)`.
```

**Review by mairaw — APPROVED** [review]
LGTM

Left a suggestion for you to consider before merging

**Comment by Thraka** [comment]
> LGTM
> 
> Left a suggestion for you to consider before merging

I actually had it written that way at first but I wasn't sure what to say "Variables" "properties" I haven't been that exposed to MSBuild. Thanks!

**Comment by mairaw** [comment]
I went to the MSBuild docs to double check 😉 
https://docs.microsoft.com/en-us/visualstudio/msbuild/common-msbuild-project-properties?view=vs-2017

**Pull request merged**